templates/new_service/terraform/application/config/review_Terrafile

```diff
@@ -0,0 +1,3 @@
+aks:
+    source:  "https://github.com/DFE-Digital/terraform-modules"
+    version: "testing"
```

Comment:
Review apps can pick up terraform-modules changes without risk, please set it to "main"